toolchain/README.md

```diff
@@ -0,0 +1,37 @@
+RISC-V GNU Compiler Toolchain with zc* support
+==============================================
+
+This is the RISC-V C and C++ cross-compiler. It supports a variety of zc extensions for RISC-V 32
+
+## Getting the sources
+
+This repository uses submodules for Newlib, GCC and Binutils. 
+
+    $ git submodule update --init --recursive
+
+## Building
+
+There is a script provided which will build all of the required elements for the toolchain, along with standard libraries
+
+    $ sh gcc.sh
```

Comment:
I think its intended to execute that script as ./gcc.sh as it has shebang bash with -xe, running it like this make would result in shell ignoring the shebang, and thus the xe flag. 

Running it without the e flag would make it harder to debug when things go wrong. 

I think it might  be a good idea to have set -xe inside the script or we change the instructions to chmod +x gcc.sh && ./gcc.sh !